### PR TITLE
*: reorganize TiDB deployment related docs

### PR DESCRIPTION
**Pull request by lilin90**
@LinuxGit @queenypingcap @CaitinChen PTAL

Via: 
- https://github.com/pingcap/docs-cn/pull/766
- https://github.com/pingcap/docs-cn/pull/770
- https://github.com/pingcap/docs-cn/pull/772

### CODE REVIEW

File: op-guide/ansible-deployment-scale.md [hunk 1]
```diff
@@ -0,0 +1,463 @@
...
+    | node7 | 172.16.10.7 | TiKV2 |
+    | node8 | 172.16.10.8 | TiKV3 |
+    | node9 | 172.16.10.9 | TiKV4 |
+        
+2. Initialize the newly added node:
+
+    ```
+    ansible-playbook bootstrap.yml -l 172.16.10.101,172.16.10.102
+    ```
+
```
Comment (CaitinChen) [inline op-guide/ansible-deployment-scale.md]: **注**：
+> 如果 `inventory.ini` 中为节点配置了别名，如 `node101 ansible_host=172.16.10.101`，执行 ansible-playbook 时 -l 请指定别名，以下步骤类似。
+> `ansible-playbook bootstrap.yml -l node101,node102`

File: op-guide/ansible-deployment-scale.md [hunk 2]
```diff
@@ -0,0 +1,463 @@
...
+      
+    4. Use `pd-ctl` to check whether the new node is added successfully:
+    
+        ```
+        ./pd-ctl -u "http://172.16.10.1:2379"
+        ```
+    
+        > **Note:** `pd-ctl` is a command used to check the number of PD nodes.
+
+5. Roll upgrade the entire cluster:
```
Comment (CaitinChen) [inline op-guide/ansible-deployment-scale.md]: -> Apply a rolling update to the entire cluster:

File: op-guide/ansible-deployment-scale.md [hunk 3]
```diff
@@ -0,0 +1,463 @@
...
+
+4. Monitor the status of the entire cluster by opening a browser to access the monitoring platform: `http://172.16.10.3:3000`.
+
+## Decrease the capacity of a TiKV node
+
+For example, if you want to remove a TiKV node (node9) with the IP address `172.16.10.9`, take the following steps:
+
+1. Remove the node from the cluster using `pd-ctl`:
+
+    1. View the store id of node9:
```
Comment (CaitinChen) [inline op-guide/ansible-deployment-scale.md]: id -> ID

File: op-guide/ansible-deployment-scale.md [hunk 4]
```diff
@@ -0,0 +1,463 @@
...
+        
+        ```
+        ./pd-ctl -u "http://172.16.10.1:2379" -d store
+        ```
+
+    2. Remove node9 from the cluster, assuming that the store id is 10:
```
Comment (CaitinChen) [inline op-guide/ansible-deployment-scale.md]: ditto

File: op-guide/ansible-operation.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+This operation stops all the components in the entire TiDB cluster in order, which include PD, TiDB, TiKV, and the monitoring components.
+
+## Clean up cluster data
+
+```
+$ ansible-playbook unsafe_cleanup_data.yml
+```
+
+This operation stops the TiDB, Pump, TiKV and PD services, and clean up the data directory of Pump, TiKV and PD.
```
Comment (CaitinChen) [inline op-guide/ansible-operation.md]: clean -> cleans

File: op-guide/ansible-operation.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+## Destroy a cluster
+
+```
+$ ansible-playbook unsafe_cleanup.yml
+```
+
+This operation stops the cluster and clean up the data directory.
```
Comment (CaitinChen) [inline op-guide/ansible-operation.md]: ditto

**Comment by lilin90** [comment]
@LinuxGit PTAL Thanks!

File: op-guide/ansible-deployment-rolling-update.md [hunk 7]
```diff
@@ -0,0 +1,126 @@
+---
+title: Upgrade TiDB Using TiDB-Ansible
+category: operations
+---
+
+# Upgrade TiDB Using TiDB-Ansible
+
+When you perform a rolling update for a TiDB cluster, the service is shut down serially and is restarted after you update the service binary and the configuration file. If the load balancing is configured in the front-end, the rolling update of TiDB does not impact the running applications. Minimum requirements: `pd*3, tidb*2, tikv*3`.
```
Comment (LinuxGit) [inline op-guide/ansible-deployment-rolling-update.md]: s/restarted/started
restart = stop + start

File: op-guide/ansible-deployment-rolling-update.md [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+
+### Perform a rolling update using Ansible
+
+- Apply a rolling update to the PD node (only upgrade the PD service)
+
+    ```
+    $ ansible-playbook rolling_update.yml --tags=pd
+    ```
+
+    When you apply a rolling update to the PD leader instance, if the number of PD instances is not less than 3, Ansible migrates the PD leader to other nodes before stopping this instance.
```
Comment (LinuxGit) [inline op-guide/ansible-deployment-rolling-update.md]: s/other nodes/other node

File: op-guide/offline-ansible-deployment.md [hunk 9]
```diff
@@ -16,25 +18,25 @@ Before you start, make sure that you have:
...
-## Install Ansible and dependencies offline in the Control Machine
-
-Take the following steps to install Ansible offline on the CentOS 7 system:
+## Step 1: Install system dependencies on the Control Machine
 
-1. Install `pip` offline.
+Take the following steps to install system dependencies on the Control Machine installed with the CentOS 7 system.
 
-    > Download the [`pip`](https://download.pingcap.org/pip-rpms.el7.tar.gz) offline installation package to the Control Machine.
+1. Download the [`pip`](https://download.pingcap.org/pip-rpms.el7.tar.gz) offline installation package to the Control Machine.
```
Comment (LinuxGit) [inline op-guide/offline-ansible-deployment.md]: need to update url, see https://github.com/pingcap/docs-cn/blob/master/op-guide/offline-ansible-deployment.md#在中控机上安装系统依赖包

**Pull request merged**